lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 494 usage aangewezen burgemeester

## Description

Made mandaat bestuursfunctie codes a bit more consistent. 
Updated the isBurgemeester getter, now this also checks if mandaat is aangewezen burgemeester, which results in the same functionality for burgemeester as aangewezen burgemeester.

## How to test

This is viewable in two places. You now should see the beleidsdomeinen section in the mandate page of an aangewezen burgemeester. Also if you have a mandataris form and you select the mandate aangewezen burgemeester, you should now see the statussen titelvoerend and aangewezen.
